### PR TITLE
Prepublishing Nudges : bottom sheet shown only once.

### LINKED ISSUE #11718: Prepublishing Nudges : Prevent the bottom sheet from appearing multiple times.
If you press the publish button quickly in succession it can cause multiple bottom sheets to be shown. This means we should check if the fragment is visible before showing it.

### PR DESCRIPTION
**Pull request by jd-alexander**
Fixes #11718

To test:
1. Create a new post. 
2. Click the Publish button multiple times in succession.
3. Realize it only shows the bottom sheet once. 

PR submission checklist:

- [x] I have considered adding unit tests where possible.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Comment by jd-alexander** [comment]
I did a merge and this PR got broken. I will be fixing the issues later today so it can be ready for review again 🙏

File: WordPress/src/test/java/org/wordpress/android/ui/posts/prepublishing/PrepublishingTagsViewModelTest.kt [hunk 1]
```diff
@@ -32,7 +32,7 @@ class PrepublishingTagsViewModelTest : BaseUnitTest() {
     @Test
     fun `when viewModel is started updateToolbarTitle is called with the tags title`() {
         var title: UiStringRes? = null
-        viewModel.updateToolbarTitle.observeForever {
```
Comment (jd-alexander) [inline WordPress/src/test/java/org/wordpress/android/ui/posts/prepublishing/PrepublishingTagsViewModelTest.kt:35]: This change was supposed to be made in a previous PR. Corrected the name of this here so the build passes.

**Comment by jd-alexander** [comment]
@zwarm this is ready for review now 😄 Thanks!

**Comment by zwarm** [comment]
Looks good to me!

**Comment by jd-alexander** [comment]
Thanks for the review @zwarm 🙏

**Pull request merged**